release-lock-file-spec.md

```diff
@@ -238,6 +242,10 @@ which can be found in the source files for this package.  Release lockfiles
     * Values **must** conform to the *Contract Type* object definition.
 
 
+Packages **should** not include abstract contracts in the *contract types*
```

Comment:
The current language is correct.  

We encourage leaving abstract contracts out of this section because:

* They allow naming collisions to occur
* inclusion of them is not useful in a release lockfile.  (Logic is as follows)

Inclusion of abstract contracts is not useful in release lockfiles because:

* If you only want to provide the interface information, you should just do so using the *real* contract and not the interface.
* Contract Types are only referenced internally from the *contract instance* which means that it is impossible for a contract instance to reference an abstract contract since abstract contracts are not deployable.

Neither of these scenarios makes inclusion of an abstract contract the correct choice when the actual contract is available.  The spec does allow inclusion of an abstract contract here but it's rarely going to be the right choice.